packages.json

```diff
@@ -1659,7 +1659,8 @@
       {
         "name": "LYUnicodeDecodeForConsole",
         "url": "https://github.com/GeekYL/LYUnicodeDecodeForConsole",
-        "description": "LYUnicodeDecodeForConsole is a plugin for Xcode to Unicode convert Chinese in the console."
+        "description": "LYUnicodeDecodeForConsole is a plugin for Xcode to Unicode convert Chinese in the console.",
+        "screenshot": "https://github.com/GeekYL/LYUnicodeDecodeForConsole/blob/master/LYUnicodeDecodeForConsoleTest.gif"
```

Comment:
It won't work to link to the github UI, we need to link to the image directly. Could you update this to:

`https://raw.githubusercontent.com/GeekYL/LYUnicodeDecodeForConsole/master/LYUnicodeDecodeForConsoleTest.gif`